cypher/cypher-docs/src/docs/graphgists/uniqueness/uniqueness.asciidoc

```diff
@@ -1,34 +1,107 @@
 [[cypher-result-uniqueness]]
-= Cypher Result Uniqueness
+= Cypher path matching
 
-While pattern matching, Neo4j Cypher makes sure to not include matches where the same graph relationship is found multiple times in a single path matching a particular pattern.
-This is called _relationship isomorphism_ and is a very effective way of reducing the result set size and preventing infinite traversals.
-To understand this better, let's consider a few alternative options for uniqueness:
+:description: Cypher path matching uses relationship isomorphis, the same relationship cannot be returned more than once in the same result record. 
```

Comment:
I think this should be an abstract, rather than a description?
```suggestion
[abstract]
--
Cypher path matching uses relationship isomorphis, the same relationship cannot be returned more than once in the same result record. 
--
```

Reply:
I will leave it as it is. It is because the antora build uses the abstract to extract a description. So the new way forward should be to remove abstracts that look ugly on the pages and insert descriptions.